modules/civicrm_rules/civicrm_rules.contact-eval.inc

```diff
@@ -144,3 +144,24 @@ function civicrm_rules_options_list() {
   $settings['groups'] = _civicrm_get_groups();
   return $settings;
 }
+
+/**
+ * Options list callback for listing of CiviCRM Tags
+ */
+function civicrm_rules_tags_list() {
+  if(!civicrm_initialize()) {
+    return array();
+  }
+  $result = civicrm_api3('tag', 'get', array(
+              'used_for' => 'civicrm_contact',
```

Comment:
@agileware-fj Francis the indenting looks off here

Reply:
@seamuslee001 I can't actually see a definitive answer on how this should be handled for function arguments in either the Drupal or CiviCRM coding standards.
Anyway, I've changed this to indent by 2 spaces for the arrays' contents only.